### DISCUSSION TITLE
Island headlands

### DISCUSSION THREAD

**Discussion by WeAreV3N0M** [Courseplay/Courseplay_FS22#2074] [Ideas]
We already have option to set headland count at the start of courses.
Could you introduce a new field generator option to set headland count around islands? Maybe with an option to start circles from inside or outside?
It would be useful for narrow machines, such as potato or sugarbeet harvesters.

**Comment by pvaiko** [comment]
Adding a separate option for the number of headlands around the islands wouldn't be too difficult, currently it is the same as the number of headlands but not more than 3. 

Start inside/outside is not something I see changing in the near future.
